### PR TITLE
Problem: rocksdb is not updated

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- update rocksdb deps
- user can update rocksdb in advance before v1.1.0 upgrade.

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ## UNRELEASED
 
 - [#1197](https://github.com/crypto-org-chain/cronos/pull/1197) tune rocksdb options to control memory consumption.
+- [#]() Update rocksdb to `v8.5.4`.
```
Comment (yihuang) [inline CHANGELOG.md:6]:
```suggestion
- [#1207](https://github.com/crypto-org-chain/cronos/pull/1207) Update rocksdb to `v8.5.4`.
```

File: nix/rocksdb.nix [hunk 2]
```diff
@@ -1,6 +1,7 @@
 { lib
 , stdenv
 , fetchFromGitHub
+, fetchzip
```
Comment (yihuang) [inline nix/rocksdb.nix:4]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,7 @@
 ## UNRELEASED
 
 - [#1197](https://github.com/crypto-org-chain/cronos/pull/1197) tune rocksdb options to control memory consumption.
+- [#1207](https://github.com/crypto-org-chain/cronos/pull/1207) Update rocksdb to `v8.5.4`.
```
Comment (yihuang) [inline CHANGELOG.md:6]:
```suggestion
- [#1207](https://github.com/crypto-org-chain/cronos/pull/1207) Update rocksdb to `v8.6.7`.
```

File: nix/rocksdb.nix [hunk 4]
```diff
@@ -21,13 +21,13 @@
 
 stdenv.mkDerivation rec {
   pname = "rocksdb";
-  version = "8.5.3";
+  version = "8.6.7";
```
Comment (mmsqe) [inline nix/rocksdb.nix:24]: https://github.com/facebook/rocksdb/compare/v8.5.3...v8.6.7

**Review by mmsqe — COMMENTED** [review]
Not sure if can build but [grocksdb](https://github.com/linxGnu/grocksdb/tree/v1.8.4) still use 8.5.3

**Comment by yihuang** [comment]
> Not sure if can build but [grocksdb](https://github.com/linxGnu/grocksdb/tree/v1.8.4) still use 8.5.3

no problem, tested locally

**Review by mmsqe — APPROVED** [review]

**Pull request merged**